### PR TITLE
RWA-1575-Clean up job added

### PR DESCRIPTION
**Pull request by serdaroguzhanakin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1575

### Change description ###

Task clean up job added for non-prod environment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by rasikasri — APPROVED** [review]

**Review by rasikasri — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/cleanup/CleanUpJobService.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+            outcomesList = deleteHistoricProcessesFromCamunda(historicCamundaTasks, serviceToken);
+
+            AtomicReference<Long> count = new AtomicReference<>(0L);
+            await()
+                .pollDelay(5, SECONDS)
+                .atMost(10, SECONDS)
+                .until(() -> {
+                    count.set(getHistoryTaskCount().getCount());
+                    return count.get() != null;
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/cleanup/CleanUpJobService.java:93]: unit test to cover?

File: src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/cleanup/CleanUpJobService.java [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+            outcomesList = deleteActiveProcessesFromCamunda(historicCamundaTasks, serviceToken);
+
+            AtomicReference<Long> count = new AtomicReference<>(0L);
+            await()
+                .pollDelay(5, SECONDS)
+                .atMost(10, SECONDS)
+                .until(() -> {
+                    count.set(getActiveTaskCount().getCount());
+                    return count.get() != null;
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/cleanup/CleanUpJobService.java:117]: unit test to cover?

File: src/main/resources/camunda/camunda-task-clean-up-query.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (BenNzewi) [inline src/main/resources/camunda/camunda-task-clean-up-query.json:1]: could we add an integration test to verify this SQL query will behave as expected?

**Review by rasikasri — APPROVED** [review]

**Review by dharmendrak — DISMISSED** [review]

**Review by mkayad — APPROVED** [review]
LGTM

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**